src/yang/ietf-tcp.yang

```diff
@@ -170,25 +171,27 @@ module ietf-tcp {
         leaf local-address {
           type inet:ip-address;
           description
-            "Local address that forms the connection identifier.";
+            "Identifies the address that is used by the local
+             endpoint for the connection.";
```

Comment:
The local-address is part of the key (connection identifier). Therefore, I would say:

"Identifies the address that is used by the local endpoint for the connection, and is one of the four elements that form the connection identifier."

Reply:
looks good to me.

Reply:
Fixed